app/src/main/java/com/kodeco/android/countryinfo/ui/components/CountryInfoRow.kt

```diff
@@ -21,7 +33,18 @@ import com.kodeco.android.countryinfo.sample.sampleCountry
 fun CountryInfoRow(
     country: Country,
     onTap: () -> Unit,
+    onStarLiked: (Country) -> Unit
 ) {
+    val color by animateColorAsState(
+        targetValue = if (country.isFavorite) Color.Yellow else Color.Black,
+        label = "Star color"
+    )
+    val startFilling by animateIntAsState(
```

Comment:
nit: star instead of _start_?

```suggestion
    val starFilling by animateIntAsState(
```